Assets/HoloToolkit/Input/Scripts/Utilities/MotionControllerVisualizer.cs

```diff
@@ -194,11 +205,6 @@ private void Application_onBeforeRender()
         }
 
 #if UNITY_WSA && UNITY_2017_2_OR_NEWER
-        private void InteractionManager_InteractionSourceUpdated(InteractionSourceUpdatedEventArgs obj)
-        {
-            StartTrackingController(obj.state.source);
```

Comment:
Do we really want to remove this?

What happens if we lose our controllers and then get them back?

Reply:
Disregard my comment. I read your issue better.  Looks like a great change!

Reply:
InteractionSourceLost and InteractionSourceDetected handle those cases correctly.